src/DayPicker.js

```diff
@@ -574,7 +574,3 @@ export default class DayPicker extends Component {
     );
   }
 }
-
-DayPicker.DateUtils = DateUtils;
-DayPicker.LocaleUtils = LocaleUtils;
-DayPicker.ModifiersUtils = ModifiersUtils;
```

Comment:
Removing these lines seems to cause the issue. Are those that were causing the UMD bug?

Reply:
I'm digging right now.